pkg/components/accesslog.go

```diff
@@ -68,6 +69,13 @@ func (c *loggingTransport) RoundTrip(r *http.Request) (*http.Response, error) {
 	if e == nil {
 		a.Status = resp.StatusCode
 		a.HTTPContentType = resp.Header.Get("Content-Type")
+		if resp.StatusCode < 200 || resp.StatusCode > 299 {
```

Comment:
nit: why do we care about 3xx status codes? if we're concerned with errors I would think we would be looking for status codes 4xx and 5xx?

Reply:
see https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#redirection_messages 3xx status codes are usually for redirection which in this case I don't think we care about

Reply:
That makes sense. Thank you for your suggestion.